### PR TITLE
Notion.so Major fixes

### PR DESCRIPTION
**Pull request by Mirm0n**
Please check the image below:
NOTION-CHANGES-2

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -13685,6 +13685,8 @@ CSS
 
 notion.so
 
+notion.so
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13689]:
```suggestion
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -13694,14 +13696,27 @@ img[alt="People using Notion"]
...
+img[alt="Books on a shelf"]
+img[alt ="Some building blocks"]
+img[alt ="A team of people collaborating"]
+.logo-limit-height
+.notranslate::after
 
 CSS
 body {
-    background-color: var(--darkreader-neutral-background) !important;
+background-color: var(--darkreader-neutral-background) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13707]:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -13694,14 +13696,27 @@ img[alt="People using Notion"]
...
 }
 .notion-divider-block div div {
-    border-bottom: 1px solid ${rgba(55, 53, 47, 0.4)} !important;
-}
+border-bottom: 1px solid ${rgba(55, 53, 47, 0.4)} !important;}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13710]:
```suggestion
    border-bottom: 1px solid ${rgba(55, 53, 47, 0.4)} !important;
}
```

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -13694,14 +13696,27 @@ img[alt="People using Notion"]
...
+[data-darkreader-inline-fill]
+{fill: #ffffff !important;}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13712]: Don't try to specify general dark reader selectors, use a selector that target the element instead.

File: src/config/dynamic-theme-fixes.config [hunk 5]
```diff
@@ -13694,14 +13696,27 @@ img[alt="People using Notion"]
...
+.header.jsx-2604928324 {background-color: var(--darkreader-neutral-background) !important;}
+.person.jsx-2691780686{
+background-color: var(--darkreader-neutral-background) !important;}
+.notranslate {  color: var(--darkreader-neutral-text) !important;}
+.notion-selectable notion-callout-block{background-color: var(--darkreader-neutral-background) !important;}
+#notion-app > div > div.notion-overlay-container.notion-default-overlay-container > div:nth-child(2) > div > div:nth-child(2) > div > div:nth-child(1) > div.notion-scroller.vertical > div.whenContentEditable > div:nth-child(4) > div > div > div.notion-selectable.notion-callout-block > div > div
+{background-color: var(--darkreader-neutral-background) !important;}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13719]: Please format these, we don't one-liners CSS in this file.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (Mirm0n) [reply src/config/dynamic-theme-fixes.config:13710]: I think that was the code that was already written before me. Do you want me to remove them or see a better fix?

File: src/config/dynamic-theme-fixes.config (hunk 5 above)
Comment (Mirm0n) [reply src/config/dynamic-theme-fixes.config:13718]: I'm going to create another pull request cause I changed a lot of the code, I also removed some of the redundant old code that was made by someone else and made it more efficient

**Pull request closed without merging**